### PR TITLE
Timestamp for PersonImage

### PR DESCRIPTION
**Pull request by VirginiaDooley**
This work 
- changes the `PersonImage` to a `TimeStampedModel`
- add a migration with default timestamps for created and updated which takes data from last queued image to populate existing `PersonImage` instances. 
- renders the timestamp in the template to help users decide if a photo should be updated (very recent photos may not need updating at all)
Screenshot 2024-07-02 at 4 50 31 PM

These changes will also lay the groundwork for a more straightforward approach to refactoring https://github.com/DemocracyClub/yournextrepresentative/pull/2386

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207697305640851

### CODE REVIEW

File: ynr/apps/people/tests/test_person_models.py [hunk 1]
```diff
@@ -47,6 +47,42 @@ def test_get_display_image_url(self):
...
+            defaults={
+                "person": person,
+                "source": "Taken from Wikipedia",
+                "copyright": "example-license",
+                "user_notes": "A photo of Tessa Jowell",
+            },
+        )
+        modified = pi.modified
+        response = self.app.get(f"/person/{person.id}", user=self.user)
+        self.assertIn(
```
Comment (VirginiaDooley) [inline ynr/apps/people/tests/test_person_models.py:81]: BST problems? I tried this again and changed the migration update and created default to `datetime.now(timezone.utc).isoformat()` but got the same error.

Comment (symroe) [reply ynr/apps/people/tests/test_person_models.py:81]: Is this working now? CI seems ok?

Comment (VirginiaDooley) [reply ynr/apps/people/tests/test_person_models.py:81]: No, I don't think so - it's a mismatch in the time.

Comment (symroe) [reply ynr/apps/people/tests/test_person_models.py:82]: I've pushed a commit that fixes this, but I'm a little confused as to why we're not getting the right timezone back from `pi.modified`. In my understanding, the model should store the timezone and return local time, but maybe that's not how it works.

**Pull request merged**